go/obscuronode/enclave/utils.go

```diff
@@ -94,3 +94,12 @@ func printTx(t nodecommon.L2Tx, txsString []string) []string {
 	}
 	return txsString
 }
+
+func contains(s []obscurocommon.TxHash, e obscurocommon.TxHash) bool {
```

Comment:
@tudor-malene Wonder if it's worth writing this as a completely generic "is x in y?" method. But no big deal.

Reply:
I think it's possible only with generics, but we can't use those yet.